lib/core/print.toit

```diff
@@ -77,3 +80,24 @@ Does not yield the currently running task.
 */
 write_on_stderr_ message/string add_newline/bool -> none:
   #primitive.core.write_string_on_stderr
+
+/**
+Print service used by $print.
+*/
+service_value_/PrintService? := null
+service_mutex_/monitor.Mutex ::= monitor.Mutex
+
+service_ -> PrintService:
```

Comment:
Is there a better way of doing this? It feels super complex and annoying.

Reply:
I think this is what you have to do now...
And this is already an improvement: https://github.com/toitware/toit/issues/1932